### ISSUE TITLE
Suggestion: API for disturbances

### ISSUE THREAD

**Issue by Bertware** [iRail/iRail#271]
NMBS provides a list of disturbances, packed as an RSS feed.
This data isn't about specific vehicles, though it is highly related to the train schedule. We could incorporate this data as an endpoint in the API.

Link: http://www.belgianrail.be/jp/sncb-nmbs-routeplanner/help.exe/nn?tpl=rss_feed&

#### Current situation
* Every app needs to pull RSS from the (slower) NMBS server
* Every app needs to handle overhead from RSS.
* Every app needs to parse RSS (XML), while all other data can be retrieved as json.

#### How it would improve

* we could parse the RSS and only return real data (similar to parsing nmbs.be for schedules). This means apps only get the actual data, without unnecessary RSS.
* we could provide both XML and Json structures, the same way as it provides them for trains and stations. This means all related data could be retrieved in json format, instead of needing to use XML only for this data.
* we could cache the json/xml responses with a TTL of 1 minute. this means that every minute, max 2 requests (1 json, 1 xml) will take 500-1000ms, all others will be served from cache. End users should get a small speed boost from this.
* This might provide an initial starting point, should we or anyone else ever want to store/analyse this data.

---

Note that this is only a suggestion and totally up for discussion. However, to me, this seems a valueable addition. While I do support and like RSS, for this purpose, it might be more useful if apps can get data that's already transformed.

---

Example data
```

<?xml version="1.0" encoding="utf-8"?>
<rss xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
<channel>
<title>Storingen</title>
<description></description>
<link>http://www.belgianrail.be/jp/sncb-nmbs-routeplanner/help.exe/nn?tpl=him_map</link>
<language>nl-be</language>
<copyright>&#169; 2013 SNCB</copyright>
<pubDate>Wed, 01 Feb 2017 12:44:00 +0100</pubDate>
<item>
<title><![CDATA[Callenelle: Schade aan de infrastructuur.]]></title>
<description><![CDATA[Schade aan de infrastructuur.<br /> Vertragingen zijn mogelijk.<br /> Onbepaalde duur van de storing.<br /> Luister naar de aankondigingen in het station.<br />]]></description>
<link>http://www.belgianrail.be/jp/sncb-nmbs-routeplanner/help.exe/nn?tpl=showmap_external&amp;messageID=18227&amp;channelFilter=timetable,twitter,rss_line_10,livemap&amp;</link>
<pubDate>Wed, 01 Feb 2017 11:16:13 +0100</pubDate>
<guid isPermaLink="false">http://www.belgianrail.be/jp/sncb-nmbs-routeplanner/help.exe/nn?tpl=him_map18227</guid>
<dc:creator></dc:creator>
</item>
</channel>
</rss>

```

**Comment by pietercolpaert** [comment]
Mind that we already do this to create a disturbance GTFS-RT feed (in protobuf, not JSON). See for example http://analytics.irail.be/ for a direct visualization of this feed available at https://gtfs.irail.be/nmbs.

**Comment by Bertware** [comment]
This has been implemented in pull request #277
